src/extension/index.activateGithubAnalyses.ts

```diff
@@ -69,13 +69,25 @@ export function getPrimaryRepository(git: API): Repository | undefined {
 // githubCodeScanningInitialAlert.
 type ConnectToGithubCodeScanning = 'off' | 'on' | 'prompt' | 'injected' | undefined;
 
+export type GitHubCodeScanningEndpoint = 'https://api.github.com';
+
 export function activateGithubAnalyses(disposables: Disposable[], store: Store, panel: Panel, outputChannel: OutputChannel) {
     disposables.push(workspace.onDidChangeConfiguration(e => {
+        if (e.affectsConfiguration('sarif-viewer.githubCodeScanningEndpoint')) {
+            const gitHubCodeScanningEndpoint = workspace.getConfiguration('sarif-viewer').get<GitHubCodeScanningEndpoint>('githubCodeScanningEndpoint');
+            sendGithubEndpointChanged(gitHubCodeScanningEndpoint === 'https://api.github.com' ? 'default' : 'true');
+        }
         if (!e.affectsConfiguration('sarif-viewer.connectToGithubCodeScanning')) return;
         const connectToGithubCodeScanning = workspace.getConfiguration('sarif-viewer').get<ConnectToGithubCodeScanning>('connectToGithubCodeScanning');
         sendGithubConfig(connectToGithubCodeScanning ?? 'undefined');
     }));
 
+
+    const connectToGithubCodeScanning = workspace.getConfiguration('sarif-viewer').get<ConnectToGithubCodeScanning>('connectToGithubCodeScanning');
+    if (connectToGithubCodeScanning === 'off') return;
+
+    const gitHubCodeScanningEndpoint = workspace.getConfiguration('sarif-viewer').get<GitHubCodeScanningEndpoint>('githubCodeScanningEndpoint') ?? 'https://api.github.com';
```

Comment:
.get lets you supply a default value - I'd prefer setting that default to `https://api.github.com` rather than performing nullish coalescing each time.